lib/bundler.rb

```diff
@@ -365,7 +363,23 @@ def git_present?
     end
 
     def ruby_version
-      @ruby_version ||= SystemRubyVersion.new
+      ruby_engine = if defined?(RUBY_ENGINE) && !RUBY_ENGINE.nil?
+        RUBY_ENGINE.dup
+      else
+        # not defined in ruby 1.8.7
+        "ruby"
+      end
+      ruby_engine_version = case ruby_engine
+                            when "ruby"
+                              RUBY_VERSION.dup
+                            when "rbx"
+                              Rubinius::VERSION.dup
+                            when "jruby"
+                              JRUBY_VERSION.dup
+                            else
+                              raise BundlerError, "RUBY_ENGINE value #{RUBY_ENGINE} is not recognized"
+      end
+      @ruby_version ||= RubyVersion.new(RUBY_VERSION.dup, RUBY_PATCHLEVEL.to_s, ruby_engine, ruby_engine_version)
```

Comment:
I would dearly love to start ripping everything out of the `Bundler` god-module, so can we move `Bundler.ruby_version` over to `Bundler::RubyVersion.current` or maybe `.system`?